src/main.mli

```diff
@@ -27,6 +29,9 @@ val external_lib_deps
   -> Build.lib_deps Path.Map.t
 val report_error : ?map_fname:(string -> string) -> Format.formatter -> exn -> unit
 
-val bootstrap : unit -> unit
-
 val find_context_exn : setup -> name:string -> Context.t
+
+(** / *)
```

Comment:
Looking at the sources of [doc-ock](http://github.com/ocaml-doc/doc-ock) it seems odoc will only recognize `(**/**)`  , but not `(** / *)`, as a stop comment. The [manual of ocamldoc](http://caml.inria.fr/pub/docs/manual-ocaml/ocamldoc.html#sec337) also claimed that the stop comment was `(**/**)` (but it might have accepted similar things, I don't know).

You probably want to change that.